beckend/index.ts

```diff
@@ -1,18 +1,23 @@
 import express from 'express';
 import router from './router/router';
+import { sequelize } from './database-models/index';
 
 // Create a new express application instance
 const app: express.Application = express();
 // parse application/json
 app.use(express.json());
 app.use(router);
 
+
 const port = process.env.PORT || 3000;
 
 app.get('/', function (req, res) {
     res.send('Hello World!');
 });
 
-app.listen(port, function () {
-    console.log('Example app listening on port 3000!');
+sequelize.sync().then(async () => {
```

Comment:
For some initialization of app, you can create bootstrap.js, initialization.js and so on... 
and move there sync with db, start up server, maybe some configurations.
`await sequelize.sync();
app.listen(port, () => console.log(`Example app listening on port ${port}!`));`